src/ResourceManager/KeyVault/Commands.KeyVault/Models/KeyAttributes.cs

```diff
@@ -65,7 +65,7 @@ public KeyAttributes()
 
         public DateTime? Updated { get; private set; }
 
-        public bool PurgeDisabled { get; private set; }
+        public string RecoveryLevel { get; private set; }
```

Comment:
Normally I'd heed your recommendation; in this case, however, the presence of this field has security implications. The server protocol does not include this field, and a default here conveys the wrong information. I would rather have the client be broken (and make a quick script change) than use incorrect information.

Furthermore, the previous PSH release was based on a non-public KV SDK; that is, this field should have been considered "pre-release" via PSH as well, but we did not find a way to mark it as such. This PSH release uses a public KV SDK, and so it is in line with the REST API spec and the service itself.

I can add this as an entry to the breaking change log..